test/core/test_normalize.py

```diff
@@ -136,7 +134,7 @@ def test_normalize_inverted_lat(self):
             chunks={'time': 1})
 
         actual = normalize_dataset(ds)
-        xr.testing.assert_equal(actual, expected)
+        xr.testing.assert_equal(actual, ds)
```

Comment:
The check of this test is now that latitude is not inverted anymore. One might argue that we don't need it anymore at all. At least, remove the lines were the 'expected' dataset is built and rename the test to 'test_lat_not_inverted'.

Reply:
I agree that the variable `expected` (lines 128-134) can be removed entirely, but I think it's still useful to keep the test itself as a definition of current behaviour, even if it's trivial. For the name I'd prefer `test_normalize_on_increasing_latitudes`, which just refers the input format rather than the expected behaviour, and means that there's one less place to update if/when we change it again. (I try to avoid/remove "inverted" where possible, because I find it ambiguous -- to me it just implies "the opposite of whatever we currently want in this context". "increasing" / "decreasing" are far clearer.)